Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "font-kit"
-version = "0.3.1"
+version = "0.4.0"
```

Comment:
I think we should keep it for a separate commit.

Reply:
Why is that?

Reply:
Well, I can split it in two anyhow.